docs/quickstart.md

```diff
@@ -22,7 +22,7 @@ you will need to provide the password for the managed node user.
 ansible-playbook playbooks/example-use/playbook.yml -u <managed_node_username> --ask-become-pass
 ```
 
-<script id="asciicast-335159" src="https://asciinema.org/a/335159.js" async data-autoplay="true" data-speed="2"></script>
```

Comment:
This is providing args to asciinema such as autoplay and speed:
https://docs.pulpproject.org/pulp_installer/quickstart/#using-the-example-playbook

Reply:
Oh! Yes! Sorry for that. I restored it as it was.

Thank you @fao89 !